01_fib/saurabhvyas9/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// Abs returns the absolute value of x.
+func Abs(x int) int {
+	if x < 0 {
+		return -x
+	}
+	return x
+}
+
+// Returns Fibonacci and Negafibonacci series
+func fib(n int) {
+	m := Abs(n)
+	a, b, neg := 0, 1, 1
+	for i := 1; i <= m; i++ {
+		if (i%2) == 0 && n < 0 {
```

Comment:
interesting approach, a little opaque for my taste - but it works!

be nice to see it without a conditional block in the loop, but that's more of a side note :)

Reply:
Will think of it. 
For now, resolving it.